### PR TITLE
Add group and subgroup

### PR DESCRIPTION
**Pull request by nikolalardev**
## Purpose of this pull request

This pull request (PR) updated the TOC.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@nikolalardev thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5127) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5127) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by osrecio** [comment]
Hi @nikolalardev can you update the template of PR with the affected pages and the motivation of this PR is to try to understand it. 😃 

Thanks!

File: guides/v2.2/extension-dev-guide/versioning/check-version.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ---
+group: php-developer-guide
+subgroup: Versioning
```
Comment (keharper) [inline guides/v2.2/extension-dev-guide/versioning/check-version.md]: Delete `subgroup`.  It's a legacy tag that's no longer needed in most topics.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**